### ISSUE TITLE
gettingStarted.mlx should be GettingStarted.mlx

### ISSUE THREAD

**Issue by michellehirsch** [mathworks/toolboxdesign#26]
Toolbox packaging has a long history of support for getting started files named GettingStarted.mlx. Additionally, all MATLAB Examples shipped by MathWorks use MixedCase naming. The recommended name should be GettingStarted.mlx

**Comment by altmany** [comment]
Regardless of whether upper or lower camel-casing is recommended (I agree it should be upper in this case), I think MATLAB should be case-agnostic for such cases, just in case the user mistyped. Perhaps display a Code Analyzer warning on the top line when opening the file in the Editor (as happens when the filename doesn't match the declared function name), but nothing more than that.

**Comment by rpurser47** [comment]
@michellehirsch, can you point to our public documentation for having examples be MixedCase?  I understand there's precedent, but it would be good to be able to point to official guidance.

**Comment by michellehirsch** [comment]
@rpurser47 we don't have any published design standards, and the forthcoming coding guidelines don't cover. All examples shipped by MathWorks follow this standard - that's the best reflection of it.

I could also point to the [Toolbox Packaging documentation](https://www.mathworks.com/help/matlab/matlab_prog/create-and-share-custom-matlab-toolboxes.html), which specifically says to use GettingStarted.mlx. But we need to resolve the issue that the recommendation of Toolbox Best Practices (GettingStarted.mlx at top level of toolbox) is not compatible with Toolbox Packaging (GettingStarted.mlx inside of doc folder). My preference would be to update the standards to match Toolbox Packaging, at least until we can update Toolbox Packaging to be more flexible. 

If you follow Toolbox Packaging requirements, users get an option to open GettingStarted.mlx upon toolbox installation, and can access it at any time from the kabob menu in the Add-Ons Side panel.

**Comment by rpurser47** [comment]
I agree that we should revise to match the way the packaging toolbox works, and move to `doc/GettingStarted.mlx`. @michellehirsch , would you update #32 to reflect this?

**Comment by rpurser47** [comment]
Premature close.  Reopen until #32 is merged.
